config/formatting.md

```diff
@@ -84,7 +84,7 @@ $ docker inspect --format "{{lower .Name}}" container
 
 {% raw %}
 ```console
-$ docker inspect --format '{{split .Image ":"}}'
+$ docker inspect --format '{{split .Image ":"}}' container
```

Comment:
Just a small suggestion to add angle brackets for the container name placeholder to make it appear more like a placeholder than a sub-command

```suggestion
$ docker inspect --format '{{split .Image ":"}}' <container>
```

Reply:
I wholeheartedly agree, just bear in mind that those changes should be made to the rest of the examples as well and not only to these one.

Reply:
oh yeah, I didn't check the rest of the doc. We can merge it like this and I can take the brackets in a follow-up.

Reply:
Should be quite quick to implement, want me to just fix it all in this PR?

Reply:
sure, if you dont mind! thank you!